pythonFiles/vscode_datascience_helpers/dataframes/vscodeDataFrame.py

```diff
@@ -3,8 +3,34 @@
 import json as _VSCODE_json
 import pandas.io.json as _VSCODE_pd_json
 
+# PyTorch and TensorFlow tensors which can be converted to numpy arrays
+_VSCODE_allowedTensorTypes = ["Tensor", "EagerTensor"]
+
+# Function that converts tensors to DataFrames
+def convertTensorToDataFrame(ten):
```

Comment:
Can we rename from `ten` to `tensor` or the like.
I thought this was a `number ten` and was confused.